aodndata/moorings/classifiers.py

```diff
@@ -275,12 +298,20 @@ def dest_path(cls, input_file):
             return cls._make_path(dir_list)
 
         fac, subfac = cls._get_facility(input_file)
-
+        is_asfs_and_rt = subfac == 'ASFS' and cls._is_realtime(input_file)
         if subfac == 'DA':
             dir_list.append(subfac)
             dir_list.append(cls._get_nc_att(input_file, 'platform_code'))
             dir_list.append(cls._get_data_category(input_file))
             dir_list.append(cls._get_product_level(input_file))
+        elif is_asfs_and_rt: # rt files with old names not migrated yet
+            cat = cls._get_old_data_category(input_file)
+            start_time = dt.strptime(cls._get_nc_att(
+                                     input_file,
+                                     'time_coverage_start'),
+                                     '%Y-%m-%dT%H:%M:%SZ')
```

Comment:
The [`_get_nc_att`](https://github.com/aodn/python-aodncore/blob/master/aodncore/pipeline/fileclassifier.py#L76) method already provides the option to return a datetime object, so this could just be:
```python
start_time = cls._get_nc_att(input_file, 'time_coverage_start', time_format=True)
```

Reply:
done